config.php

```diff
@@ -135,11 +135,11 @@
  * You shouldn't need to change this section - but if you find this needs modification to get it working,
  * then please contribute what you did back to the docker-dev repository :)
  */
-if (!empty($_SERVER['HTTP_X_ORIGINAL_HOST']) && strpos($_SERVER['HTTP_X_ORIGINAL_HOST'], 'ngrok.io') !== false) {
+if (!empty($_SERVER['HTTP_X_ORIGINAL_HOST']) && (strpos($_SERVER['HTTP_X_ORIGINAL_HOST'], 'ngrok.io') !== false || strpos($_SERVER['HTTP_X_ORIGINAL_HOST'], 'ngrok-free.app') !== false)) {
```

Comment:
Do we think ngrok might ever start using any other URLs in the future.

Maybe its work adding an `$ngrokUrls` array we can amend when necessary, and this line can become:
```suggestion
if (in_array($_SERVER['HTTP_X_ORIGINAL_HOST'], $ngrokUrls)) {
```

Reply:
Thanks @jonnybarnes. changed it to be more easily maintainable